.github/workflows/pytest.yml

```
@@ -14,6 +14,21 @@ on:
 jobs:
   test:
     runs-on: ubuntu-latest
+    permissions:
+      actions: none
+      attestations: none
+      checks: none
+      contents: read
+      deployments: none
+      id-token: none
+      issues: none
+      discussions: none
+      packages: none
+      pages: none
+      pull-requests: none
+      repository-projects: none
+      security-events: none
+      statuses: none      
```

```suggestion
      statuses: write      
```
Might also need this permission 
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/collaborating-on-repositories-with-code-quality-features/about-status-checks

---

Nevermind, maybe not. https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/collaborating-on-repositories-with-code-quality-features/about-status-checks#:~:text=GitHub%20Actions%20generates%20checks%2C%20not%20commit%20statuses%2C%20when%20workflows%20are%20run.